types/stylelint/index.d.ts

```diff
@@ -157,6 +157,7 @@ declare namespace stylelint {
 		ruleSeverities: { [ruleName: string]: RuleSeverity };
 		customMessages: { [ruleName: string]: RuleMessage };
 		ruleMetadata: { [ruleName: string]: Partial<RuleMeta> };
+		fixersData: { [ruleName: string]: Array<FixerData> };
```

Comment:
[ask] Where is `fixersData` used? If autofix is performed only in `report()`, `fixersData` seems no longer needed.

Reply:
I won't ask you to read all our exchanges with romain but this comment might be relevant : https://github.com/stylelint/stylelint/pull/7730#discussion_r1632050103

Reply:
I understand your thoughts. We can go on anyway and return later if needed since `fixersData` is just internal.